ada/ast.py

```diff
@@ -14695,6 +14695,11 @@ class EntrySpec(BaseSubpSpec):
     )
     returns = Property(No(T.TypeExpr.entity))
 
+    xref_equation = Property(Entity.family_type.then(
+        lambda r: r.sub_equation,
+        default_val=LogicTrue())
+    )
```

Comment:
The closing parenthesis for `then(` should be on line 14701 as well since it was opened on the same line as the one for `Property(`.